dolt_integrations/dvc/examples/fashion-mnist/dolt-dvc-rewrite.md

```diff
@@ -0,0 +1,314 @@
+
+# Introduction
+
+"Git for Data" and "Data Version Control" are colloquially
```

Comment:
You need to mention Dolt in here by name, and explain how it is related to "Git for Data"